Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.15 as builder
```

Comment:
In Readme golang version is mentioned as 1.14. Should we use the same?

Reply:
since, we haven't tested in 1.15, lets keep it to 1.14.

Reply:
#42

Reply:
Can we specify minimum golang version needed ?

Reply:
For initial release lets close this PR with go 1.14 and in the next release lets upgrade to 1.16

Reply:
Okay cool